locales/templates/app.pot

```diff
@@ -1446,6 +1446,45 @@ msgctxt "logo_text"
 msgid "Service provided by"
 msgstr ""
 
+#. Recommended news onboarding title and content.
+msgctxt "recommended_news_title"
+msgid "Recommended News For You"
+msgstr ""
+
+#, c-format
+msgctxt "recommended_news_content"
+msgid ""
+"Tap “Yes” to begin seeing news recommended for you by our AI news engine."
+" You can turn off recommendations and control your settings anytime in "
+"News-&gt;Settings. %1$s"
```

Comment:
The placeholder should be explained in the localization comment.

Reply:
@Conjuror 
Change
#. Recommended news onboarding title and content.
to
#. Recommended news onboarding title and content. The %1$s here is a placeholder for "Learn more" which directs users to SUMO page with detail information about the feature.

Reply:
DONE